src/views/set2/Set2.vue

```diff
@@ -365,10 +273,7 @@
             self.d3.json(self.publicPath + "data/NHDWaterbody_DelawareBay_pt6per_smooth.json"),
             self.d3.json(self.publicPath + "data/reservoirs.json"),
             self.d3.json(self.publicPath + "data/dams.json"),
```

Comment:
Couldn't find where `public/data/DRB_Wshed_1per_smooth.json` was referenced (is a deleted file). I assume it wasn't in the code and deleting that file was part of clean up?

Reply:
Likewise `Segments_subset_4per_smooth_pt5miBuffer_symdiff.json`, `matrix_obs.csv`, `narrative_rectangle.json`, and `observed_reach_station_coords.json`

Reply:
Yes - Sorry, I should have been more clear. While I was going through and deleting the files associated with the ch1 stand-in map I realized there were some data files that I had used in earlier drafts but were not currently being used, so I deleted those as well.